ui/src/styles/globals.css

```diff
@@ -7,6 +7,7 @@ body {
   font-family: PT Sans Narrow, sans-serif;
   margin: 0;
   color: #333;
+  line-height: 1.43;
```

Comment:
🤔  do we really need this? maybe they just did it randomly in the legacy one

Reply:
It does make it the text look better and more like legacy so I would still add it